hexrd/ui/mask_manager_dialog.py

```diff
@@ -390,10 +391,18 @@ def show_masks(self):
         fig.canvas.draw_idle()
         fig.show()
 
-    def hide_masks(self, checked):
-        if checked:
-            HexrdConfig().visible_masks.clear()
-        else:
-            HexrdConfig().visible_masks = list(self.masks.keys())
+    def toggle_all_masks(self):
```

Comment:
This is a great function! But I am changing the name to `update_visibility_checkboxes` instead. The name `toggle_all_masks` makes me thing it is going to go through the checkboxes and do this: `cb.setChecked(not cb.isChecked())` (which is what toggling is).